server/server/server.go

```diff
@@ -66,10 +66,14 @@ type Server struct {
 	api             *api.API
 }
 
-func New(cfg *config.Configuration, singleUserToken string, db store.Store, logger *mlog.Logger, serverID string) (*Server, error) {
+func New(cfg *config.Configuration, singleUserToken string, db store.Store,
+	logger *mlog.Logger, serverID string, wsAdapter ws.Adapter) (*Server, error) {
```

Comment:
Nit: when we get to line wrapping func arguments it's probably time to pass a struct.